contracts/0.8.9/LidoLocator.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.8.9;
+
+import {ILidoLocator} from "../common/interfaces/ILidoLocator.sol";
+
+/**
+ * @title LidoLocator
+ * @author mymphe
+ * @notice Lido service locator
+ * @dev configuration is stored as public immutables to reduce gas consumption
+ */
+contract LidoLocator is ILidoLocator {
```

Comment:
Rename to ContractsLocator or LidoContractsLocator? Because of clashing of "Lido" as protocol and "Lido" as Lido.sol contracts

Reply:
I am for `LidoLocator`

Reply:
I've pondered and for LidoLocator at the end. Just because it shorter and sounds sharper.